.travis.yml

```diff
@@ -1,27 +1,28 @@
+dist: xenial
 sudo: false
 language: python
-python: "3.5"
-env:
-  - TOX_ENV=py27
-  - TOX_ENV=py34
-  - TOX_ENV=py35
 matrix:
   include:
+    - python: 3.4
+      env: TOX_ENV=py34
+
+    - python: 3.5
+      env: TOX_ENV=py35
+
     - python: 3.6
       env: TOX_ENV=py36
-  include:
+
     - python: 3.7
       env: TOX_ENV=py37
-  include:
     - python: 3.7
       env: TOX_ENV=lint
-  include:
+
     - python: 2.7
-      env: TOX_ENV=lint
```

Comment:
it's generally a good idea to run lint on py2 and py3

Reply:
Ah, yeah, I'll add that back to the matrix, thanks.